python/list-streams/README.md

```diff
@@ -0,0 +1,68 @@
+# Kinesis Video Streams Counter Lambda Function
+
+This AWS Lambda function fetches the total number of Kinesis Video Streams within a specified AWS region using the AWS SDK for Python (Boto3). It employs a retry mechanism for handling transient errors and provides error handling for potential exceptions.
+
+## Functionality
+
+The Lambda function performs the following tasks:
+
+1. Connects to the Kinesis Video Streams service in the specified AWS region.
+2. Retrieves the list of Kinesis Video Streams using the `list_streams` API.
+3. Iterates through the paginated responses to count the total number of streams.
+4. Provides retry logic for transient errors such as throttling or service limits.
+5. Returns the total count of Kinesis Video Streams.
+
```

Comment:
Do we need to  add a note that the total counts should be retrievable within a single Lambda run...

Reply:
I think it's self-explanatory but sure we can add it.